sechub-doc/src/docs/asciidoc/documents/code2doc/client/02_sechub_client.adoc

```diff
@@ -131,7 +133,40 @@ action `unmarkFalsePositives`
 include::sechub_client_falsepositve_list_example_unmark.json[]
 ----
 
-==== Options
+===== interactiveMarkFalsePositives
+Instead of creating a json file to mark false positives, you can also do this interactively:
+Go to the project directory containing `sechub.json` and a report file from a recent scan and run `sechub interactiveMarkFalsePositives`.
+
+You can select the report file using the `-file` option of the client. Otherwise the latest report file in the current directory is automatically selected.
+
+For each finding in the reports file, you are prompted: `"Add this as false positive?"`
+Your choices are:
+
+- `y` Yes - Mark as false positive. You are prompted for an optional comment line which can help later to understand why this has been marked.
```

Comment:
I would change the word order to: `which can later help to understand`

Reply:
Fixed